cmake/Modules/Findblis.cmake

```diff
@@ -0,0 +1,42 @@
+SET(BLIS_INCLUDE_SEARCH_PATHS
+       /usr/local/include/blis
+)
+
+SET(BLIS_LIB_SEARCH_PATHS
+       /usr/local/lib
+)
```

Comment:
One could probably add more search paths here, akin to the OpenBLAS one:
https://github.com/ROCmSoftwarePlatform/pytorch/blob/fcda9d1f1d2ff80865936aad7a064cebfe48d4fa/cmake/Modules/FindOpenBLAS.cmake#L16

Reply:
We have made the necessary changes.